desktop/apps/categories2/components/__snapshots__/AppContainer.spec.js.snap

```diff
@@ -1,7 +1,109 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`renders correctly 1`] = `
-<h1>
-  The Art Genome Project
-</h1>
+<div>
+  <div
+    style={
+      Object {
+        "backgroundColor": "orange",
```

Comment:
Hmm, this is a funny looking representation of the inline styles — I wonder if this is what Jest snapshots always look like for these?